README.md

```diff
@@ -198,18 +158,26 @@ MicroMasters and Open-Discussions share a cookie, which means that they need to
 This can be emulated by modifying your hosts file, for example:
 
  ```
-192.168.99.100  micromasters.odl.local
-192.168.99.101  od.odl.local
+127.0.0.1  mm.odl.local
+127.0.0.1  od.odl.local
  ```
 
-The following variable should also be set in your MicroMasters ``.env`` file:
-
-  - ``OPEN_DISCUSSIONS_JWT_SECRET``: same value as in the open-discussions ``.env`` file
-  - ``OPEN_DISCUSSIONS_COOKIE_NAME``: same value as in the open-discussions ``.env`` file
-  - ``OPEN_DISCUSSIONS_COOKIE_DOMAIN``: the root domain, for example ``odl.local``, should match correspding variable in Ope-Discussions
-  - ``OPEN_DISCUSSIONS_REDIRECT_URL``: the domain-based URL for open-discussions,
-    for example ``http://od.odl.local:8063/``
-  - ``OPEN_DISCUSSIONS_BASE_URL``: the base internal URL for open-discussions,
-    i.e. ``http://192.168.99.101:8063`` or ``http://localhost:8063/``
-  - ``FEATURE_OPEN_DISCUSSIONS_POST_UI=True``
-  - ``FEATURE_OPEN_DISCUSSIONS_CREATE_CHANNEL_UI=True``
+The following variables should also be set in your Open-Discussions `.env` file, assuming you have modified your `hosts` file using `mm.odl.local` for Micromasters and `od.odl.local` for Open-Discussions:
+
+  - `MICROMASTERS_EXTERNAL_LOGIN_URL=http://mm.odl.local:8079/discussions/`
+  - `MICROMASTERS_BASE_URL=http://docker.for.mac.localhost:8079/` (OSX)
+  - `MICROMASTERS_BASE_URL=http://<docker-accessible-internal-ip>:8079/` (non-OSX)
+  - `OPEN_DISCUSSIONS_COOKIE_DOMAIN=odl.local`
```

Comment:
Let's simplify this markdown:

```
OPEN_DISCUSSIONS_COOKIE_DOMAIN=odl.local
MICROMASTERS_EXTERNAL_LOGIN_URL=http://mm.odl.local:8079/discussions/
### Linux users should use this value...
MICROMASTERS_BASE_URL=http://mm.odl.local:8079/
### OSX users should use this value...
MICROMASTERS_BASE_URL=http://docker.for.mac.localhost:8079/
```

I also changed the `MICROMASTERS_BASE_URL` value for Linux. I'm about 90% sure that will be the only IP value they need